homeTask_7/homeTask_7/Comparators/FnameLnameLengthCompare.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace homeTask_7.Comparators
+{
+    class FnameLnameLengthCompare : IComparer<Employee>
+    {
+        public int Compare([AllowNull] Employee x, [AllowNull] Employee y)
+        {
+            return  (x.Name.Length + x.LastName.Length) - (y.Name.Length + y.LastName.Length);
```

Comment:
In my opinion better way create Fullname property in Employee class and do operation with it.
`public string Fullname {get; set;} = Firstname + Lastname;`

Reply:
> public string Fullname {get; set;} = Firstname + Lastname;

Done